pkgs/development/interpreters/python/hooks/pytest-check-hook.sh

```diff
@@ -0,0 +1,24 @@
+# Setup hook for pytest
+echo "Sourcing pytest-check-hook"
+
+pytestCheckPhase() {
+    echo "Executing pytestCheckPhase"
+    runHook preCheck
+
+    args=" -m pytest"
+    if ! [ -z "$disabledTests" ]; then
+      args="$args -k disabledTests"
```

Comment:
I like this one. We should have the same for nosetest.

Reply:
Yes. Note it's not yet correct. Need to implement the "and not <test>" bit. Here I dislike having to write these things in bash.

Reply:
Wasn't there a similar implementation but in nix for this?
Something like having an attribute `disabledTests` but as an argument to `buildPython*`.

Reply:
That was https://github.com/NixOS/nixpkgs/pull/53142
I prefer to do as much as possible in Nix, however, that does not always work. In this case, the problem was that the generated value (a string) cannot be overridden.